packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -19,12 +19,28 @@ import '../globals.dart' as globals;
 import '../migrations/cmake_custom_command_migration.dart';
 import 'visual_studio.dart';
 
+// These characters appear to be fine: @%()-+_{}[]`~
+const String badCharacters = r"'#!$^&*=|,;<>?";
```

Comment:
Style is to prefix constants with `k` and to keep things private if possible:

```suggestion
const String _kBadCharacters = r"'#!$^&*=|,;<>?";
```

See: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k